packages/create-remix/templates/arc-stack/app.arc

```diff
@@ -0,0 +1,24 @@
+@app
+remix-aws-stack
+
+@http
+/*
+  method any
+  src server
+
+@static
+
+@aws
+region us-east-1
+
+@tables
+people
+  pk *String
+
+notes
+  pk **String
```

Comment:
in DynamoDB-land, `pk` generally means partition or primary key, and `sk` is for sort key, but they're defined backwards here. This is more than likely going to confuse people, especially considering the `people` table is using `pk` as the partition key.